frontend/appflowy_flutter/pubspec.yaml

```diff
@@ -46,68 +46,63 @@ dependencies:
   appflowy_editor:
     # path: /Users/lucas.xu/Desktop/appflowy-editor
     git:
-      url: https://github.com/LucasXu0/appflowy-editor.git
-      ref: 25eb16
+      url: https://github.com/Xazin/appflowy-editor.git
+      ref: e2cf098
```

Comment:
This should be changed to the correct ref on https://github.com/AppFlowy-IO/appflowy-editor/ once the upgrade on the editor is merged.